configs/config/benchmark/linear_image_classification/datasets/clevr_count.yaml

```diff
@@ -0,0 +1,50 @@
+# @package _global_
```

Comment:
looks great! would it make sense to move these out to
`linear_image_classification/clever/eval_linear_clevr_count.yaml` ? that will create more visibility to the task! we can reference any example paper that uses this benchmark. thoughts? : )

Reply:
Added in 5faf4a95557f16572e76d7f26ab1525e11fe1933 (I reach the same performance as documented in the CLIP paper)